packages/Box/Box.jsx

```diff
@@ -43,7 +43,7 @@ const inlineBetweenStyles = props =>
     { between: props.between, inline: props.inline },
     ({ between, inline }, breakpoint) => {
       const base = {
-        display: between ? 'flex' : 'block',
+        display: typeof between !== 'undefined' ? 'flex' : 'block',
```

Comment:
Could we not just do `display: between !== undefined ? ‘flex’ : ‘block’`? (Drop the `typeof`)